docker/diffkemp-devel/run-container.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
+import os
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--build-dir")
+parser.add_argument("--diffkemp-dir", default=".")
+parser.add_argument("--llvm-version", default="13")
```

Comment:
I wonder how difficult it'd be to set the default LLVM version to the latest currently supported version so that we don't have to change this with every LLVM release.

Reply:
I removed the default option, transferring the choice to Dockerfile, where the default LLVM version is also specified in a [variable](https://github.com/zacikpa/diffkemp/blob/container-autobuild/docker/diffkemp-devel/Dockerfile#L3). It suppose it's OK to keep that one, since we need to change that file with every LLVM release nonetheless.